### ISSUE TITLE
Bump brewcask dependency to 0.0.7

### ISSUE THREAD

**Issue by hyleung** [boxen/our-boxen#824]
With `brewcask` 0.0.6, I'm seeing the following error:

```
Error: Execution of 'brew boxen-latest brew-cask' returned 1: Error: No available formula with the name "brew-cask"
Error: /Stage[main]/Brewcask/Package[brew-cask]/ensure: change from absent to present failed: Execution of 'brew boxen-latest brew-cask' returned 1: Error: No available formula with the name "brew-cask"
```

I think this is due to an issue addressed by https://github.com/boxen/puppet-brewcask/pull/46, which was included in `brewcask` 0.0.7.

I've made this change in a couple of my own environments and it seems to get rid of the error. Haven't verified on a completely clean environment yet tho'.

**Comment by jabley** [comment]
👍 I've done the same, and works for me.

https://github.com/jabley/our-boxen/pull/9

**Comment by jacobbednarz** [comment]
fixed in #825.
